build/patches/JIT-less-toggle.patch

```diff
@@ -107,13 +107,15 @@ diff --git a/content/public/android/java/src/org/chromium/content/browser/Device
  import org.chromium.base.StrictModeContext;
  import org.chromium.content_public.common.ContentSwitches;
  import org.chromium.ui.base.DeviceFormFactor;
-@@ -22,4 +23,9 @@ public class DeviceUtilsImpl {
+@@ -22,4 +23,11 @@ public class DeviceUtilsImpl {
              }
          }
      }
 +    public static void addJitlessSwitch() {
```

Comment:
This method is not called from anywhere?

Reply:
It's called here:
https://github.com/bromite/bromite/blob/79035ef630515542be4ded6138ab12735b044d1a/build/patches/JIT-less-toggle.patch#L95